### DISCUSSION TITLE
1Panel 的 WAF 检测域名时会区分大小写

### DISCUSSION THREAD

**Discussion by Radar5404** [1Panel-dev/1Panel#5213] [Q&A]
### 1Panel 版本

 v1.10.9-lts

### 问题描述

使用1panel创建网站时，如果域名包含大小写字母，如 XXy.xx时，浏览器无法访问该网站，会出现“网站不存在，请检查域名”的界面。
在”网站-域名设置”里将域名改为全小写的 xxy.xx即可正常访问。
1Panel 的 waf 会检测网站是否在 1Panel 的网站列表中，如果访问的域名不在，就会拦截。所以猜测浏览器访问时域名不区分大小写，但waf检测域名时区分了大小写，所以检测到域名不在列表中。

### 重现步骤

如上

**Comment by Radar5404** [comment]
感觉是不是应该在自动生成网站配置文件时，在 server_name部分自动转化为小写，如 server_name xxy.xx;

**Comment by zhengkunwang223** [comment]
建议添加的时候使用小写 WAF 匹配是根据 server_name 匹配的 如果你的 server_name 和 WAF 在请求中取到的名字不一致 就会拦截
